### PR TITLE
Added cyro to starcraft 2

### PR DESCRIPTION
**Pull request by Havaticis**
Ok I think I go it right this time. :D

### CODE REVIEW

File: StarCraft2.ahk [hunk 1]
```diff
@@ -131,11 +138,14 @@ class SC2
         this.tankyPos := yPos
         this.casterxPos := xPos + 188    ; compared to the center: a little bit to the right
         this.casteryPos := yPos - 120    ; compared to the center: a little bit upper
+        this.cryoxPos := xPos - 20    ; compared to the center: a little bit to the right
```
Comment (TheBestPessimist) [inline StarCraft2.ahk]: compared to the center: a little bit to the `left`

Comment (Havaticis) [reply StarCraft2.ahk]: That's so the Cyro throws the Cyro missiles slightly behind him, keeping the zombies close enough to get hit by most of everything.

**Comment by TheBestPessimist** [comment]
Thanks a lot for adding this, and also putting up with all my complaining about everything!

_(quite a lot of people actually asked for adding cryo, but i never did it since i don't play it)_

**Comment by Havaticis** [comment]
No I understand. It's something that you do very often and take pride in, so I don't blame you for being nit picky.  And yeah I know a lot of people were asking for it (I was one of them), but I just decided to make it.  :D  Thanks for all the help you provided.

**Pull request merged**